packages/flutter_tools/lib/src/build_runner/web_fs.dart

```diff
@@ -629,7 +629,6 @@ class BuildDaemonCreator {
         if (testTargets.hasBuildFilters) {
           b.buildFilters.addAll(testTargets.buildFilters);
         }
-        return b;
```

Comment:
@yjbanov this is the only change here that I'm not confident in. I couldn't quickly figure out what happened to this return value.

Reply:
I only rewrote it from the `=>` notation to the `{}` block notation and automatically kept the existing semantics. I didn't check that `DefaultBuildTarget` took a `void` function. So your change looks fine.